src/QA_Components/AddQuestion.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Button} from '@material-ui/core';
+
+class AddQuestion extends React.Component {
```

Comment:
Just want to call attention to the fact we're mixing and matching classes and functional components. I'm not sure if it matters too much